### DISCUSSION TITLE
Feature: Show first verified URL in people search

### DISCUSSION THREAD

**Discussion by Ciantic** [mastodon/mastodon#21045] [General]
Type a famous person in search, and try to find which one is which? I suggest to change it to this:

Notice "hypercritical.co", I mocked it to here:

<img src="https://user-images.githubusercontent.com/64731/202800088-6a8dc052-18e4-425c-a460-2d5114d678fb.png" width=400 />

Take first *verified* URL from user profile and show it in the search, simple, and easier for all. Most of the people is known for something verified.

**Comment by davetapley** [comment]
Another good example of this as mentioned [here](https://mastodon.me.uk/@geoffl/111175890423090807).

BBC are testing fediverse, but only one of their official profiles shows up, and is buried below bots 😑 

<img width="888" alt="Search_for_bbc_news_-_Urbanists_Social" src="https://github.com/mastodon/mastodon/assets/48232/0d5bc6cc-c477-4ebd-b492-6e01a8f6cde7">

**Comment by shaedrich** [comment]
I would also really love to be able to see verified profiles when doing at mentions, similarly to how you see them in the search results, because when doing at mentions, there's currently no way to see the real verified profile.

| Search results | at-mentions currently | at-mentions perhaps in the future |
| -------------- | --------------------- | --------------------------------- |
| ![grafik](https://github.com/user-attachments/assets/cbfa2d18-98ab-4a61-a4ef-7a16ca1677ee) | ![grafik](https://github.com/user-attachments/assets/ca5b660d-73c0-4de6-9bb7-5581f9d58aa8) | ![grafik](https://github.com/user-attachments/assets/f7ab7f90-032d-4c97-aa3e-d38ec505a8bc) |
